CommitChange/houdini
# remove most ramda from components

split out from https://github.com/CommitChange/houdini/pull/964

to test
- make a donation, make sure info step works, with address autocomplete, make sure card form works correctly, etc.
- edit recurring donation, change amount, update address
- go to /admin, make sure search works
- go to a campaign and an event with activity feed not hidden, make sure it shows as expected
- check that an event and a campaign can be copied/duplicated
- check that gift options or ticket levels can be reordered by dragging
- go to a supporter, add a note, then delete it, make sure delete confirmation modal works
